quickstarts/quarkus-with-s2i.yaml

```diff
@@ -0,0 +1,198 @@
+        quadrant of the **quarkus-quickstarts** deployment represents the Git
+        repo off the associated code.  The icon shown can be for Bitbucket,
+        GitHub, GitLab or generic Git.  
+
+        1. Click on the icon to navigate to the associated Git repository.
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: >-
+          Verify you can see the code associated with the sample app:
+
+          - Was a new browser tab opened with
+          https://github.com/quarkusio/quarkus-quickstarts?
+      summary:
+        failed: Try the steps again.
+        success: You accessed the git repo associated with your deployment.
+      title: View the associated Git repo
+    - description: >-
+
+        To view the pod status:
+        
+        1. Click the browser tab running OpenShift.  
+        
+            - Notice that the **quarkus-quickstarts** deployment has a pod donut imposed on the
+        circle, representing the pod status (i.e. blue = running).  
+            - The color of the donut indicates
+        the pod status.  
+
+        1. Hover over the pod donut to see the pod status in a tooltip.
+
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: |-
+          To verify you see the pod status:
+
+          Do you see the number of associated pods and their statuses?
+      summary:
+        failed: Try the steps again.
+        success: Your deployment has one running pod.
+      title: View the pod status
+    - description: >-
+        Since a Java builder image was used during the Import from Git flow, a
+        Java icon is used by default in topology.  
+        
+          - The icon is determined by the
+        value of the `app.openshift.io/runtime` label. 
+        
+          - Let’s change the icon
+        associated with the quarkus-quickstarts deployment to indicate that it
+        is a Quarkus application. 
+
+        1. Click on the **quarkus-quickstarts** deployment 
+
+        1. In the side panel, click on the **Actions** drop-down and click the
+        **Edit labels** menu item. 
+
+        1. Click on the “x” next to the `app.openshift.io/runtime=java` label to
+        remove the entry 
+
+        1. Add an additional label `app.openshift.io/runtime=quarkus`
+
+        1. Click on **Save** to save the labels
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: >-
+          To verify you have updated the icon :
+
+          Do you now see the quarkus icon rather than a java icon in the
+          **quarkus-quickstarts** deployment? 
+      summary:
+        failed: Try the steps again.
+        success: You changed the icon associated with your deployment.
+      title: Change the deployment icon to Quarkus
+    - description: >-
+        Now let's Open the Quarkus application URL!  
```

Comment:
Open --> open?

Reply:
I'm going to change this sentence across the board.